libraries/chain/protocol_feature_manager.cpp

```diff
@@ -363,6 +363,17 @@ host function call will trigger a transition to the Savanna consensus algorithm.
               builtin_protocol_feature_t::disable_deferred_trxs_stage_2
             }
          } )
+         (  builtin_protocol_feature_t::sync_call, builtin_protocol_feature_spec{
+            "SYNC_CALL",
+            fc::variant("49f6fbc4fee045bc5c8b09a96b2b5c096afdc320ac29b54e0f3805c660b88483").as<digest_type>(),
+            // SHA256 hash of the raw message below within the comment delimiters (exclude newline after /*) (do not modify message below).
+/*
+Builtin protocol feature: SYNC_CALL
+
+Enables synchronous calls to functions in other and own contracts.
+*/
+            {}
```

Comment:
I think this should depend on `builtin_protocol_feature_t::savanna`. If we ever mint a new genesis we want to be able to know that this can't be enabled without all the other protocol features.

Reply:
Thanks. I was thinking about the dependency on SAVANNA but thought it could work standalone. But it is safer to have all previous protocol features enabled. Will add the dependency.